src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,111 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Line extends BaseEntity{
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    private String name;
+
+    @Column(nullable = false)
+    private String color;
+
+    @ManyToOne
+    @JoinColumn(name = "UPSTATION_ID")
+    private Station upStation;
+
+    @ManyToOne
+    @JoinColumn(name = "DOWNSTATION_ID")
+    private Station downStation;
+
+    @Column(nullable = false)
+    private Long distance;
+
+    public Line() {
+    }
```

Comment:
JPA에서 entity의 경우 지연로딩일때 프록시객체로 활용되어서 기본 생성자가 필요하고 상속때문에 접근제한자를 public, protected 로 설정 가능한데 아무래도 최소화해주는게 좋을 것 같네요 :)
```suggestion
    protected Line() {
    }
```

Reply:
네 알겠습니다. Entity 객체를 생성시 기본 생성자를 protected 하도록 하겠습니다.
조언 감사합니다.

```Station 도 같이 변경하였습니다!!```